src/components/templates/Camp/components/Map.tsx

```diff
@@ -109,495 +90,156 @@ export const Map: React.FC<MapProps> = ({
-                  height: height + "%",
-                }}
-                key={room.title}
-                onClick={() => {
-                  if (!IS_BURN) {
-                    openModal(room);
-                  } else {
-                    setRoomClicked((prevRoomClicked) =>
-                      prevRoomClicked === room.title ? undefined : room.title
-                    );
-                  }
-                }}
-                onMouseEnter={() => {
-                  dispatch(retainAttendance(true));
-                  setRoomHovered(room);
-                }}
-                onMouseLeave={() => {
-                  dispatch(retainAttendance(false));
-                  setRoomHovered(undefined);
-                }}
-              >
-                <div
-                  className={`camp-venue ${
-                    roomClicked === room.title ? "clicked" : ""
-                  }`}
-                >
-                  <div
-                    className={`grid-room-btn ${
-                      isUnderneathRoom && "isUnderneath"
-                    }`}
-                  >
-                    <div
-                      className="btn btn-white btn-small btn-block"
-                      onClick={(e) => onJoinRoom(e, room)}
-                    >
-                      {venue.joinButtonText ?? "Join now"}
-                    </div>
-                  </div>
-                  <div className="camp-venue-img">
-                    <img
-                      src={room.image_url}
-                      title={room.title}
-                      alt={room.title}
-                    />
-                  </div>
-                  {venue.roomVisibility === RoomVisibility.hover &&
-                    roomHovered &&
-                    roomHovered.title === room.title && (
-                      <div className="camp-venue-text">
-                        <div className="camp-venue-maininfo">
-                          <div className="camp-venue-title">{room.title}</div>
-                          <CampAttendance
-                            attendances={attendances}
-                            venue={venue}
-                            room={room}
-                          />
-                        </div>
-                      </div>
-                    )}
-
-                  <div className={`camp-venue-text`}>
-                    {(!venue.roomVisibility ||
-                      venue.roomVisibility === RoomVisibility.nameCount ||
-                      (venue.roomVisibility === RoomVisibility.count &&
-                        hasAttendance)) && (
-                      <div className="camp-venue-maininfo">
-                        {(!venue.roomVisibility ||
-                          venue.roomVisibility ===
-                            RoomVisibility.nameCount) && (
-                          <div className="camp-venue-title">{room.title}</div>
-                        )}
-                        <CampAttendance
```

Comment:
this was refactored into `src/components/templates/Camp/components/MapRoomsOverlay.tsx`